packages/meso-js/src/createPostMessageBus.ts

```diff
@@ -43,6 +60,9 @@ export const createPostMessageBus = (
   if (getParentWindowOrigin() === targetOrigin) {
     // target is parent, currently in iframe
     postMessageWindow = window.parent;
+  } else if (syntheticWindowMessageHandle) {
```

Comment:
Nit: would it make sense for we to put this as the first `if` predicate check? That way we can skip `getParentWindowOrigin()` logic if we know there's a window being passed in.

Reply:
Done. Good call